.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -57,3 +62,109 @@ jobs:
           git config --global user.email "broadbot@broadinstitute.org"
           git commit -am "Auto update cromwell-helm chart version to $NEXT_VERSION_GCP and coa-helm chart version to $NEXT_VERSION_COA"
           git push https://broadbot:$BROADBOT_TOKEN@github.com/broadinstitute/cromwhelm.git main
+      
+      - name: Find out latest chart version from cromwhelm repository
```

Comment:
Is this step different from `- name: Find out next chart version`? Should it become more like a "check the expected new chart has appeared in the charts directory" kind of an action?

Reply:
> Should it become more like a "check the expected new chart has appeared in the charts directory" kind of an action?

It's only different in that it reads directly from the charts, rather than using the computed version number

I originally added it as part of a copy/paste from the other workflow, and saw the similarity with the earlier step, but then decided to keep it because it seemed useful for that reason alone... but I agree that it'd be even better to perform a check against the computed number.